tests/atf_python/sys/net/vnet.py

```diff
@@ -180,6 +180,8 @@ def create_iface(self, alias_name: str, iface_name: str) -> List[VnetInterface]:
 
     @staticmethod
     def is_autodeleted(iface_name: str) -> bool:
+        if iface_name == "lo0":
+            return False
         iface_type = re.split(r"\d+", iface_name)[0]
         return iface_type in IfaceFactory.AUTODELETE_TYPES
```

Comment:
I think the AUTODELETE_TYPES thing is not a great idea in general. It requires the test framework to maintain a list of interface types which will get stale over time. I'm not sure why it doesn't include `if_bridge` or `if_wg` for example.

Is the intent here to destroy all interfaces belonging to a vnet before destroying the vnet itself?

Reply:
> I think the AUTODELETE_TYPES thing is not a great idea in general. It requires the test framework to maintain a list of interface types which will get stale over time. I'm not sure why it doesn't include `if_bridge` or `if_wg` for example.

Probably because there are no tests that need these types at the moment.

> Is the intent here to destroy all interfaces belonging to a vnet before destroying the vnet itself?

Yes, similarly to what is done in atf-sh (https://github.com/freebsd/freebsd-src/commit/80fc25025ffcb0d369fc0b6d4d272ad6fd3f53c3)

Reply:
Hmm, ok. The problem that motivated commit 80fc250 is now fixed, so I'm still kind of skeptical, but alright.

Reply:
Ahh.. OK, I believe @AlexanderChernikov's words were at the time:
> There is a known problem resulting from a corner case in the kernel delayed object reclamation model.

I'll try test destroying everything from the host... but I remember half-epairs remaining and something similar to https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270628 when testing with VLANs.